spec/systems/view_keyword_list_spec.rb

```diff
@@ -109,16 +106,107 @@
         click_button(I18n.t('auth.login'))
       end
 
-      attach_file('csv_import_form[file]', file_path)
-
-      click_button(I18n.t('keyword.upload'))
-
       expect(current_path).to eql(keywords_path)
       expect(page).to have_selector('table')
 
-      within 'table' do
-        within 'tbody' do
-          expect(page).to have_selector('tr', count: 50)
+      within 'table tbody' do
+        expect(page).to have_selector('tr', count: 50)
+      end
+    end
+
+    context 'given in_queue status' do
+      it 'displays spinning icon' do
+        user = Fabricate(:user)
+        Fabricate(:keyword, user_id: user[:id], keyword: 'Eden Hazard')
+
+        visit keywords_path
+
+        within 'form' do
+          fill_in('username', with: user[:username])
+          fill_in('password', with: 'password')
+
+          click_button(I18n.t('auth.login'))
+        end
+
+        expect(current_path).to eql(keywords_path)
+        expect(page).to have_selector('table')
+
+        visit keywords_path
+
+        within 'table tbody' do
+          expect(page).to have_selector('tr', count: 1)
+
+          tr_list = all('tr')
+
+          expect(tr_list[0]).to have_selector('.fa-spinner')
+        end
+      end
+    end
+
+    context 'given completed status' do
+      it 'displays information icon' do
+        user = Fabricate(:user)
+        keyword = Fabricate(:keyword, user_id: user[:id], keyword: 'Eden Hazard')
+
+        VCR.use_cassette('with_top_position_adwords', record: :none) do
+          GoogleScrapingJob.perform_now(keyword.id, keyword.keyword)
+        end
+
+        visit keywords_path
+
+        within 'form' do
+          fill_in('username', with: user[:username])
+          fill_in('password', with: 'password')
+
```

Comment:
how about moving this to a `login` method as when the app is growth, we have to repeat this many time.....

Reply:
Fixed in 2b87ded7a19954df2cf41eeab89f8988cf59dccb